### ISSUE TITLE
how to use,in windows romote control

### ISSUE THREAD

**Issue by wangxue1990** [hikaricai/p2p_tun#1]
p2pclient -b ":23443" -r "*.*.*.*:4000" -l ":10022" -t "127.0.0.1:3389"  -k 4321
is this right?

**Comment by hikaricai** [comment]
yes its right, and now we do not need to use "-b" para, client will use a random udp port .
